sdk/storage/azblob/blockblob/client.go

```diff
@@ -165,6 +165,20 @@ func (bb *Client) Upload(ctx context.Context, body io.ReadSeekCloser, options *U
 	return resp, err
 }
 
+// PutBlobFromURL - The Put Blob from URL operation creates a new Block Blob where the contents of the blob are read from
+// a given URL. Partial updates are not supported with Put Blob from URL; the content of an existing blob is overwritten
+// with the content of the new blob. To perform partial updates to a block blob’s contents using a source URL, use the Put
+// Block from URL API in conjunction with Put Block List.
+// For more information, see https://learn.microsoft.com/rest/api/storageservices/put-blob-from-url
+func (bb *Client) PutBlobFromURL(ctx context.Context, copySource string, options *PutBlobFromURLOptions) (PutBlobFromURLResponse, error) {
```

Comment:
`CopyBlobFromURL` is a better name here

Reply:
`CopyBlobFromURL` sounds like this, https://learn.microsoft.com/en-us/rest/api/storageservices/copy-blob-from-url.
[Put Blob](https://learn.microsoft.com/en-us/rest/api/storageservices/put-blob) operation is named [`Upload`](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/storage/azblob/blockblob/client.go#L156). Similarly we can name [Put Blob from URL](https://learn.microsoft.com/en-us/rest/api/storageservices/put-blob-from-url) as `UploadBlobFromURL`.

Reply:
Done!